src/frontend/hooks/useHentTilbakekrevingsbehandlinger.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useHttp } from '@navikt/familie-http';
+
+import { hentTilbakekrevingsbehandlinger } from '../api/hentTilbakekrevingsbehandlinger';
+
+export const HentTilbakekrevingsbehandlingerQueryKeyFactory = {
+    fagsak: (fagsakId: number | undefined) => ['tilbakekrevingsbehandlinger', fagsakId],
```

Comment:
Tror ikke denne trenger å ta i mot `undefined`

```suggestion
    fagsak: (fagsakId: number) => ['tilbakekrevingsbehandlinger', fagsakId],
```